labelme/app.py

```diff
@@ -95,6 +96,13 @@ def __init__(
         self.flag_dock.setWidget(self.flag_widget)
         self.flag_widget.itemChanged.connect(self.setDirty)
 
+        self.label_flag_dock = self.label_flag_widget = None
+        self.label_flag_dock = QtWidgets.QDockWidget('Label Flags', self)
+        self.label_flag_dock.setObjectName('Label Flags')
+        self.label_flag_widget = QtWidgets.QListWidget()
+        self.label_flag_dock.setWidget(self.label_flag_widget)
+        self.label_flag_widget.itemChanged.connect(self.labelFlagChanged)
```

Comment:
These widget and dock are not necessary anymore, right?

Reply:
It is a pop up when you add the new label and it is a dock too.

Reply:
![image](https://user-images.githubusercontent.com/17500670/57486754-62bf7c80-7274-11e9-9b7a-f525014d66e9.png)
![image](https://user-images.githubusercontent.com/17500670/57486771-6f43d500-7274-11e9-9c72-b75029fa17eb.png)
![image](https://user-images.githubusercontent.com/17500670/57486809-85519580-7274-11e9-8f36-5d9afc363631.png)

Reply:
I think the dock for flags is not necessary, and label flags should be included to the `labelDialog`.

What do you think?

Reply:
I think it's useful to have them in both. But the config.yaml file can make it not be shown if I'm not mistaken or you could hit the "x" button to close that dock.

Reply:
I think it is useful in some times, but I'm not sure if it is worth to show in the main window.
For you, it is really useful? And you always select flag in the main window, not in the popup window?

Reply:
I want to know in what situation you need to select a flag in the MainWindow.

Reply:
When modifying flags, I use the dock because it is faster.

Reply:
Ok. In that case I think we should avoid including this because it is
inconsistent to how we modify the label (e.g. person, dog). There is no way
of modifying the label of polygon in the main window.